lib/main_common.dart

```diff
@@ -88,8 +91,15 @@ Future<void> _firebaseInitializer(FlavorName flavorName) async {
 }
 
 void _activateManagers() {
+  activateConnectionManager();
   activateRemoteConfig();
   if (!kIsWeb) {
     activateCrashReportIfEnabled();
   }
 }
+
+void activateConnectionManager() {
+  final ConnectionStatusManager connectionStatus =
+      DependecyInjection.instance.get<ConnectionStatusManager>();
+  connectionStatus.activateConnectionManager();
+}
```

Comment:
Do not put this method here. 
Declare it directly in the manager and import the file to use the method in main